appmap/test/test_django.py

```diff
@@ -99,23 +99,36 @@ def user_post_view(_request, username, post_id):
 def echo_view(request):
     return django.http.HttpResponse(request.body)
 
+def user_included_view(_request, username):
+    return django.http.HttpResponse(f'user {username}, included')
+
 urlpatterns = [
-    django.urls.path('test', view),
-    django.urls.path('', view),
-    django.urls.re_path('^user/(?P<username>[^/]+)$', user_view),
-    django.urls.path('post/<int:post_id>', post_view),
-    django.urls.path('post/<username>/<int:post_id>/summary', user_post_view),
-    django.urls.re_path(r'^post/unnamed/(\d+)$', post_unnamed_view),
-    django.urls.path('echo', echo_view),
+    path('test', view),
+    path('', view),
+    re_path('^user/(?P<username>[^/]+)$', user_view),
+    path('post/<int:post_id>', post_view),
+    path('post/<username>/<int:post_id>/summary', user_post_view),
+    re_path(r'^post/unnamed/(\d+)$', post_unnamed_view),
+    path('echo', echo_view),
+    re_path(r'^post/(included)/', include([
```

Comment:
Why the parentheses?

Reply:
I believe they're leftover from some debugging. I'll double-check. If they are actually accomplishing something, I'll add a comment to indicate what. Otherwise, I'll remove them.

Reply:
Didn't need them, took them out.